plugin/vimkubectl.vim

```diff
@@ -7,13 +7,16 @@ command -bar -bang -complete=custom,vimkubectl#allResources -nargs=? Kget call v
 command -bar -bang -complete=custom,vimkubectl#allNamespaces -nargs=? Kns call vimkubectl#switchOrShowNamespace(<q-args>)
 command -bar -bang -complete=custom,vimkubectl#allContexts -nargs=? Kctx call vimkubectl#switchOrShowContext(<q-args>)
 command -bar -bang -complete=custom,vimkubectl#allResourcesAndObjects -nargs=+ Kedit call vimkubectl#editResourceObject(<q-args>)
+command -bar -bang -complete=custom,vimkubectl#allResources -nargs=+ Kdoc call vimkubectl#viewResourceDoc(<q-args>)
```

Comment:
We could do more clever completion support by also suggesting resource fields, but we can do this as a follow up, don't have to do it now.

Reply:
Autocomplete for the resource fields would be really awesome! I was recently asking the kubectl people about a feature like that for `kubectl explain`: https://github.com/kubernetes/kubectl/issues/1557#issue-2128602708

I'm not aware of any tools that support that kind of advanced querying on the resource manuals but some of that might fit nicely with vimkubectl.